app/src/main/res/values/styles.xml

```diff
@@ -215,7 +215,7 @@
         <!-- Close button icon -->
         <item name="closeIcon">@drawable/ic_close_black_24dp</item>
         <!-- Search button icon -->
-        <item name="searchIcon">@drawable/ic_search_black_24dp</item>
+        <item name="searchIcon">@drawable/ic_search_white_24dp</item>
```

Comment:
Err why did you remove it

Reply:
@abishekvashok, sorry but you are addressing the drawable file or styles.xml

Reply:
Styles.xml

Reply:
@abishekvashok, if the drawable file ic_search_black_24dp is removed then referencing it in styles.xml will produce error.  And drawable file ic_search_black_24dp must be deleted as it is currently not used anywhere in the project.

Reply:
The drawable was being used. You removed it's use and deleted it. See the entire diff

Reply:
@abishekvashok, I had to remove the use of the drawable file because of this

![screenshot_20190119-102052](https://user-images.githubusercontent.com/25201519/51422539-669c9580-1bd6-11e9-9549-f4fb72bf689a.jpg)


So I had to create ic_search_white_24dp